### PR TITLE
Fix GrapGapMatch interaction response

### PR DESCRIPTION
**Pull request by krampstudio**
Related to https://oat-sa.atlassian.net/browse/TAO-3110

Requires https://github.com/oat-sa/extension-tao-itemqti/pull/802 (should be merged at the same time to ensure consistency).

### CODE REVIEW

File: views/js/qtiCommonRenderer/renderers/interactions/GraphicGapMatchInteraction.js [hunk 1]
```diff
@@ -352,6 +352,7 @@ define([
 
                     // ... or removing the existing gapfiller
                     } else {
+                        debugger;
```
Comment (ssipasseuth) [inline views/js/qtiCommonRenderer/renderers/interactions/GraphicGapMatchInteraction.js]: debugger

Comment (krampstudio) [inline views/js/qtiCommonRenderer/renderers/interactions/GraphicGapMatchInteraction.js]: oops

**Review by ssipasseuth — APPROVED** [review]
Ok it works. Waiting the apporval of the related PR https://github.com/oat-sa/extension-tao-itemqti/pull/802 and https://github.com/oat-sa/extension-tao-itemqti/pull/806

**Comment by ssipasseuth** [comment]
But I am not sure that https://github.com/oat-sa/extension-tao-itemqti/pull/806 is required as https://github.com/oat-sa/extension-tao-itemqti/pull/806 fixes match interaction whereas this one fixes the graphic match interaction ...

**Comment by krampstudio** [comment]
@ssipasseuth you're right. It's a mistake. #806 has nothing related to those PRs.

**Comment by krampstudio** [comment]
A new PR will come soon

**Pull request closed without merging**